Packs/MicrosoftCloudAppSecurity/Integrations/MicrosoftCloudAppSecurity/MicrosoftCloudAppSecurity.yml

```diff
@@ -0,0 +1,852 @@
+category: Authentication
+commonfields:
+  id: MicrosoftCloudAppSecurity
+  version: -1
+configuration:
+- display: Server URL (e.g. https://example.net)
+  name: url
+  required: true
+  type: 0
+- display: User's key to access the API
+  name: token
+  required: true
+  type: 4
+- display: Fetch incidents
+  name: isFetch
+  required: false
+  type: 8
+- display: Incident type
+  name: incidentType
+  required: false
+  type: 13
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+- display: Incident severity
+  hidden: false
+  name: severity
+  options:
+  - Low
+  - Medium
+  - High
+  - All
+  required: false
+  type: 16
+- display: Maximum alerts to fetch
+  hidden: false
+  name: max_fetch
+  required: false
+  type: 0
+- additionalinfo: First fetch timestamp (<number> <time unit>, e.g., 12 hours, 7 days)
+  display: First fetch time
+  hidden: false
+  name: first_fetch
+  required: false
+  type: 0
+- display: Incident resolution status
+  hidden: false
+  name: resolution_status
+  options:
+  - Open
+  - Dismissed
+  - Resolved
+  - All
+  required: false
+  type: 16
+- additionalinfo: 'A custom filter by which to filter the returned files. If you pass the custom_filter argument it will override the custom_filter parameter from the integration instance configuration. For more information about filter syntax, see the (Microsoft documentation)[https://docs.microsoft.com/en-us/cloud-app-security/api-activities#filters].'
```

Comment:
```suggestion
- additionalinfo: 'A custom filter by which to filter the returned files. If you pass the custom_filter argument it will override the other filters from the integration instance configuration. For more information about filter syntax, see the (Microsoft documentation)[https://docs.microsoft.com/en-us/cloud-app-security/api-activities#filters].'
```